test/testProject.js

```diff
@@ -169,14 +169,17 @@ module.exports.project = {
                                 '      <trans-unit id="1" resname="foobar" restype="plural" datatype="x-android-resource" extype="one">\n' +
                                 '        <source>%d friend commented</source>\n' +
                                 '        <target state="new">%d friend commented</target>\n' +
+                                '        <note>{"pluralForm":"one","pluralFormOther":"foobar"}</note>\n' +
```

Comment:
regarding this type of comment, what kind of value is returning if I use `getComment()`?

Reply:
It returns a string still, same as before. It is the stringified json. It is up to the caller to do JSON.parse() to turn it into a json object.